src/app/pages/settings/settings.page.html

```diff
@@ -1,20 +1,33 @@
 <ion-header>
-    <ion-toolbar>
-        <ion-buttons slot="start">
-            <ion-menu-button></ion-menu-button>
-        </ion-buttons>
-        <ion-title>
-            Settings
-        </ion-title>
-    </ion-toolbar>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>
+      Settings
+    </ion-title>
+  </ion-toolbar>
 </ion-header>
 
 <ion-content padding>
   <ion-list>
     <h4>Push Notification</h4>
     <ion-item>
-        <ion-label item-left>Enabled</ion-label>
-        <ion-toggle [(ngModel)]="pushEnabled" (ionChange)="togglePushEnabled();" item-right></ion-toggle>
+      <ion-label item-left>Enabled</ion-label>
+      <ion-toggle
+        [(ngModel)]="pushEnabled"
+        (ionChange)="togglePushEnabled()"
+        item-right
+      ></ion-toggle>
+    </ion-item>
+    <h4>Simulate offline in Task page</h4>
```

Comment:
To simplify review we may need to unstage this for the moment.
I think we are not ready yet to get both ideas merged. We could work on this in separate PR.